test/rst_doc.md

```diff
@@ -468,27 +473,27 @@ Return Types
     .. php:method:: returnClassFromSameNamespace()
 
         :returns: An object instance of a class from the same namespace.
-        :returntype: OtherLibrary\\ReturnedClass
+        :returntype: ReturnedClass
 
     .. php:method:: returnClassFromOtherNamespace()
 
         :returns: An object instance of a class from another namespace.
-        :returntype: LibraryName\\SubPackage\\SubpackageInterface
+        :returntype: \\LibraryName\\SubPackage\\SubpackageInterface
 
     .. php:method:: returnClassConstant()
 
         :returns: The value of a specific class constant.
-        :returntype: LibraryName\\NamespaceClass::NAMESPACE_CONST
+        :returntype: \\LibraryName\\NamespaceClass::NAMESPACE_CONST
 
     .. php:method:: returnGlobalConstant()
 
-        :returns: The value of a specific global constant.
+        :returns: The value of a specific global constant. # TODO link is not working without "\\"
```

Comment:
What's up here?

Reply:
`:returntype: SOME_CONSTANT` return type reference with constant is unresolved (has no clickable link) - I need to find a solution first. If ok, let's merge this big PR and these small TODOs can be fixed later, probably together with #56.